pom.xml

```diff
@@ -1265,7 +1264,6 @@
       <dependency>
         <groupId>org.apache.jackrabbit</groupId>
         <artifactId>jackrabbit-core</artifactId>
-        <version>${jackrabbit-core.version}</version>
         <scope>test</scope>
       </dependency>
```

Comment:
This is unnecessary given that you're moving dependency management to the parent POM in https://github.com/pentaho/maven-parent-poms/pull/172